CONTENT/schedule.markdown

```diff
@@ -38,11 +38,11 @@ We currently organize 2 sessions per week. Every magic session accepts **up to 1
 <img alt="" border="0" src="https://www.paypalobjects.com/en_US/i/scr/pixel.gif" width="1" height="1">
 </form>
 
-**If you can't afford to pay the full price, we are proud to be able to offer you our Pay-what-you-can program, supported by [our amazing donors](...).**
+**If you can't afford to pay the full price, we also offer Pay-what-you-can program, supported by [our amazing donors](...).**
```

Comment:
I think it's without a or the, but we have to give it to someone to proof read it anyway...